css-structure/selectors-and-cascading/css/styles.css

```diff
@@ -50,93 +50,79 @@ nav li {
 
 /* 1. These styles should apply to the headline */
 h1 {
-  outline: 2px dashed red;
   font-size: 2rem;
   margin: 0 2rem 0 0;
   color: #2673bb;
 }
 
 /* 2. These styles should apply to all links */
 a {
-  outline: 2px dashed red;
   color: #ff00e6;
 }
 
 /* 3. These styles should apply to all external links (you can use the attribute selector) */
 a[href^="http"] {
-  outline: 2px dashed red;
   color: #ff8000;
 }
 
 /* 4. These styles should apply to all links in the main navigation */
 nav a {
-  outline: 2px dashed red;
   color: #ffffff;
   text-decoration: none;
 }
 
 /* 5. These styles should apply to the main navigation link corresponding to the current page */
 nav a.current {
-  outline: 2px dashed red;
   color: #22e781;
 }
 
 /* 6. These styles should apply to all links in the footer */
 footer a {
-  outline: 2px dashed red;
   color: inherit;
 }
 
 /* 7. These styles should apply to all paragraphs and all lists (please use a class selector) */
 .list,
 .text {
-  outline: 2px dashed red;
   color: #2c3f3f;
 }
 
 /* 8. These styles should apply to all "notes" */
 .text-node {
-  outline: 2px dashed red;
   font-size: 0.8em;
   color: #808080;
 }
 
 /* 9. These styles should apply to the welcome text */
 #welcome-text {
-  outline: 2px dashed red;
   font-size: 1.2rem;
   color: #000000;
 }
 
 /* 10. These styles should apply to the paragraph in the footer */
 footer p.text {
-  outline: 2px dashed red;
   color: #ffffff;
 }
 
 /* 11. These styles should apply to all lists */
 ul {
-  outline: 2px dashed red;
   padding: 1rem 2rem;
   background: #caced2;
 }
 
 /* 12. These styles should apply to all list items */
 li {
-  outline: 2px dashed red;
   margin: 0 0 0.5rem 0;
 }
 
 /* 13. These styles should apply to all nested lists */
 ul ul {
```

Comment:
Are you sure this works? I used li ul li

Reply:
Hm, that's interesting. :slightly_smiling_face: I guess it is also a bit a question what `These styles should apply to all nested lists` exactly means.

For sure, your version and mine _are different_, because in your version the rule will be applied to `li` elements and in my version to `ul` elements.

Cool, thanks! :+1: